docs/cloud/cloud-secrets.md

```diff
@@ -22,148 +20,105 @@ This document covers the use of cloud-hosted secrets. It builds upon the underst
 
-Alice and Bob sign up for earthly accounts using alice@example.com and bob@example.com respectively:
+When secrets need to be referenced in an Earthfile, you need to declare the project the secrets belong to at the top of the Earthfile, after the `VERSION` declaration.
 
-```bash
-earthly account register --email alice@example.com --token ...
-earthly account register --email bob@example.com --token ...
+```Dockerfile
+VERSION 0.7
+PROJECT <org-name>/<project-name>
 ```
 
-Alice then creates an organization called hush-co:
-
-```bash
-earthly org create hush-co
-```
+Then, cloud secrets can be referenced in a similar way to [locally-defined secrets](build-args.md).
 
-Alice then creates a secret under the `project-zulu` sub directory:
+For example:
 
-```bash
-earthly secrets set /hush-co/project-zulu/transponder-code peanut
+```Dockerfile
+RUN --secret MY_KEY=my_key echo $MY_KEY
 ```
 
-Alice then grants Bob read permission on all of `project-zulu`:
+The env variable `MY_KEY` will be set with the value stored under the secret key `my_key`.
 
-```bash
-earthly org invite /hush-co/project-zulu/ bob@example.com
-```
-
-Bob now has permission to everything under the `/hush-co/project-zulu/` directory. If he runs
+Or, to reference a user secret:
 
-```bash
-earthly secrets ls /hush-co/
+```Dockerfile
+RUN --secret MY_KEY=/user/my_private_key echo $MY_KEY
 ```
 
-he will see:
+## Security Details
 
-```
-/hush-co/project-zulu/transponder-code
-```
+The Earthly command uses HTTPS to communicate with the cloud secrets server. The server encrypts all secrets using OpenPGP's implementation of AES256 before storing it in a database. We use industry-standard security practices for managing our encryption keys in the cloud. For more information see our [Security page](https://earthly.dev/security).
 
-However if Alice were to create any secrets outside of `project-zulu`, Bob would not be able to list or retrieve them.
+## Migrating from the old 0.6 experimental version of Earthly secrets
 
-## Using cloud secrets in CI
+The 0.6 version of Earthly cloud secrets is no longer supported. To help migrate to the new version, a migration command has been made available.
 
-To reference secrets from a CI environment, you can make use of the password or ssh-key authentication referenced under the login/logout section, or you can generate an authentication token by running:
+In the 0.6 version, the secrets were stored globally as part of an Earthly organization. In the new version, secrets are stored per project. To migrate, you need to first create a new project using `earthly` 0.7+ (if you haven't already), and then run the migration command.
+
+You will first need to have read access to the source organization, and write access to the project you will be migrating to. The source organization can be the same as the destination one.
 
 ```bash
-earthly account create-token [--write] <token-name>
+earthly project --org <org-name> create <project-name>
```

Comment:
should there also be an example command here for creating the organisation?

Reply:
actually. probably not, since this is just for migrating secrets, which assumes one already has an org.